cloud-provider-snapshot-restore-jobs/docs/links.md

```diff
@@ -0,0 +1,40 @@
+# MongoDB::Atlas::CloudProviderSnapshotRestoreJobs Links
```

Comment:
I think we should remove this files from the repo, and add it to the gitignore, we can autogenerate them and ship them with the binaries.